base/doc/ltnews41.tex

```diff
@@ -294,12 +295,31 @@ \section{Configurable output routine}
    of \LaTeX~2.09; thus for
    compatibility we continue to support this behavior.
 \end{description}
+
+By default the separation between the last line of text and the
+footnotes (\cs{skip}\cs{footins}) is not measured from the baseline of
+the last text line but from its bottom. This goes back to plain \TeX{}
+where it is done in the same way.  Similarly, \cs{textfloatsep} is
+added between text and bottom floats not starting from the baseline of
+the last text line. Typographically speaking this is suboptimal
+because it means that with \cs{flushbottom} in effect, the position of
+the last textline when followed by footnotes or floats depends on
+whether or not it contains characters with descenders.
+
+For this reason there are now the sockets
+\socket{build/column/footnotes/baselineattach} and
+\socket{build/column/floats/baselineattach}. The can be assigned the
+plug \plug{on} in which case the attachment of footnotes/floats is
+measured from the baseline of the last text line. To mimic the
+previous behavior, they are by default both \plug{off}.  One probably
+wants to set them both to \plug{on} or both to \plug{off}, but for
+maximum flexibility the two can be individually changed.
+
```

Comment:
I (think) I'd lean towards combining these.

Reply:
it sure is possible to merge the too and in fact I'm also leaning towards that since everything else seems artifical, but without feedback I thought I stay with both.